File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task33_CLAUDE_claude-3-5-sonnet-20240620.java

import java.util.PriorityQueue;

public class Solution {
    public static int queueTime(int[] customers, int n) {
        PriorityQueue<Integer> tills = new PriorityQueue<>(n);
        
        for (int i = 0; i < n; i++) {
            tills.offer(0);
        }
        
        for (int customerTime : customers) {
            int tillTime = tills.poll();
            tills.offer(tillTime + customerTime);
        }
        
        int maxTime = 0;
        while (!tills.isEmpty()) {
            maxTime = tills.poll();
        }
        
        return maxTime;
    }
}

File: dataset/output/CLAUDE_claude-3-5-sonnet-20240620/Task33_CLAUDE_claude-3-5-sonnet-20240620.cpp

#include <algorithm>
#include <vector>

class Solution {
public:
    static int queueTime(std::vector<int> customers, int n) {
        std::vector<int> tills(n, 0);
        for (int customer : customers) {
            tills[0] += customer;
            std::sort(tills.begin(), tills.end());
        }
        return *std::max_element(tills.begin(), tills.end());
    }
};
